c/src/ml-api-service-offloading.c

```diff
@@ -1184,3 +1184,20 @@ ml_service_offloading_request (ml_service_h handle, const char *key,
     nns_edge_data_destroy (data_h);
   return ret;
 }
+
+/**
+ * @brief Register new information, such as neural network models or pipeline descriptions, on a offloading server.
```

Comment:
The contents of the brief are different from ml-api-service-offloading.h line 103.
Did you intend to?

Reply:
I added comment, same contents w/ml_service_offloading_request() function.